pkgs/development/python-modules/faraday-plugins/default.nix

```diff
@@ -40,20 +41,10 @@ buildPythonPackage rec {
   ];
 
   checkInputs = [
+    faraday
```

Comment:
This module requires [Fraday (Open Source Vulnerability Management Platform)](https://github.com/infobyte/faraday) not FaradayRF.